src/sentry/static/sentry/app/views/settings/account/accountDetails.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import {addErrorMessage, addSuccessMessage} from 'app/actionCreators/indicator';
+import AsyncView from 'app/views/asyncView';
+import AvatarChooser from 'app/components/avatarChooser';
+import Form from 'app/views/settings/components/forms/form';
+import JsonForm from 'app/views/settings/components/forms/jsonForm';
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+import accountDetailsFields from 'app/data/forms/accountDetails';
+import accountPreferencesFields from 'app/data/forms/accountPreferences';
+import {APIRequestMethod} from 'app/api';
+import {t} from 'app/locale';
+
+const ENDPOINT = '/users/me/';
+
+class AccountDetails extends AsyncView {
+  getEndpoints(): ReturnType<AsyncView['getEndpoints']> {
+    return [['user', ENDPOINT]];
+  }
+
+  renderBody() {
+    const {user} = this.state;
+    const {location} = this.props;
+
+    const formCommonProps: Partial<Form['props']> = {
+      apiEndpoint: ENDPOINT,
+      apiMethod: 'PUT' as APIRequestMethod,
```

Comment:
@priscilawebdev Do you get a type check failure if you don't have this type casting?

Reply:
yes